### DISCUSSION TITLE
All people is my INTANCE , helpp!! :3

### DISCUSSION THREAD

**Discussion by Rusbelito** [TheLastBen/fast-stable-diffusion#1197] [Q&A]
I am trying to train a model in colab with the face of my daughter, but when training it and riding it in SD and for example put snoop dog, my daughter comes out with the face of snoop, and so all I put is my daughter with another face, how can I solve it? Try unchecking "`Remove_existing_instance_images`" but it stays the same, can you please help me?

**Comment by chewtoys** [comment]
(myimagename:1.5)

keep lowering the number until you get the desired style you want, and dont go over 2.0

**Reply by Rusbelito** [reply]
i am using this colab . where i could change this option that you said me 
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb#scrollTo=LC4ukG60fgMy
I don't see it

**Reply by chewtoys** [reply]
actual promp when you test the image, not colab

**Reply by Rusbelito** [reply]
I don't understand you, am I leaving an empty field?
[https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb#scrollTo=LC4ukG60fgMy](url)

**Reply by chewtoys** [reply]
photo of (myimagename:1.5) sitting on chair, oil painting, cinematic lighting

in AUTOMATIC1111/stable-diffusion-webui

if your getting overfitting, then you over trained your model, or if your model is not showing up, you under trained it

try training the model with text encoder of 300, and make sure your images are named right, XdFGhmyimage (1).png, make sure myimage has random words, not name of your subject, name the Session_Name same as your image name so you remember.

10 images

650 2e5
300

then disconnect, terminate session, reconnect, do not download model, Load a Session

resume
800 5e6
0

check Save_Checkpoint_Every_n_Steps
200
200

you will have 4 models, download and test each one.
